tasks/Trivia Multiple Choice Quiz.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+> Participants were organized into groups of four and performed a series of tasks in a series of rounds with old and new group members. 
+
+Tasks were creative, intellective and cognitive conflict tasks. 
+
+Creative Task: collaborators wrote a short online text advertisements for product campaigns for Kickstarter, teams decided on a final ad and one member submits on behalf of the team 
+
+Intellective task: participants were asked to correctly answer a series of questions without looking up the solutions online, instead debating with their group members on what the answers to the questions could be. At the end teams decided on the correct answer for each prompt and one person submitted a list on behalf of the team 
+
+Cognitive Conflict Task: teams were given $500,000 and had to allocate funds between a number of competing programs which were each designed to appeal to particular personal values
+
+
+# References
+> Tasks based on: Joseph Edward McGrath. 1984. Groups: Interaction and performance. Vol. 14. Prentice-Hall, Englewood Cliffs, NJ
+
+> Creative Stimuli based on:
+Niloufar Salehi, Andrew McCabe, Melissa Valentine, and Michael Bernstein. 2017. Huddler: Convening Stable and Familiar Crowd Teams Despite Unpredictable Availability. (2017), 1700–1713. https://doi.org/10.1145/2998181.2998300
+
+> Steven P Dow, Alana Glassco, Jonathan Kass, Melissa Schwarz, Daniel L Schwartz, and Scott R Klemmer. 2010. Parallel prototyping leads to better design results, more divergence, and increased self-efficacy. ACM Transactions on Computer-Human Interaction (TOCHI) 17, 4 (2010), 18
+Viability Scale: Jessica Nicole Cooperstein. 2017. Initial Development of a Team Viability Measure. (2017)
+
+
+# Procedure
+## Steps
+> 1. Participants assigned to task group
```

Comment:
Maybe include that the teams were allocated based on social golfer matching, so that each collaboration
consists of members who have never interacted before.